packages/tenant-process/src/model/domain/apiConverter.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  ExternalId,
+  Tenant,
+  TenantKind,
+  tenantKind,
+  TenantAttribute,
+  TenantVerifier,
+  TenantRevoker,
+  TenantMail,
+  TenantMailKind,
+  tenantMailKind,
+  TenantFeature,
+} from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { z } from "zod";
+import * as api from "../generated/api.js";
+import {
+  ApiExternalId,
+  ApiTenantFeature,
+  ApiTenantKind,
+  ApiTenantAttribute,
+  ApiTenantVerifier,
+  ApiTenantRevoker,
+  ApiMail,
+  ApiMailKind,
+} from "./models.js";
+
+export function tenantKindToApiTenantKind(input: TenantKind): ApiTenantKind {
```

Comment:
Let's update all the converters with the new naming, for example this one will be `toApiTenantKind`